samples/single-page-app/README.md

```diff
@@ -2,8 +2,7 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) using the "single-page-app" template. It was then "ejected" so that the configuration files are locally editable.
 
-To enable the @fullstory/babel-plugin-annotate-react plugin we edited the `fullstory-babel-plugin-annotate-react/samples/single-page-app/config/webpack.config.js` file. Since this sample is using the local plugin we use `../..` as the plugin name (around line 376) but you should use the npm package name of `@fullstory/babel-plugin-annotate-react`.
-
+To enable the @fullstory/babel-plugin-annotate-react plugin edit your module rules for files matching `/\.(js|mjs|jsx|ts|tsx)$/` to include the fullstory plugin. In our sample repository we modified the `fullstory-babel-plugin-annotate-react/samples/single-page-app/config/webpack.config.js` file (around line 426) Since this sample is using the local plugin we use `../..` as the plugin name but you should use the npm package name of `@fullstory/babel-plugin-annotate-react`.
```

Comment:
```suggestion
To enable the @fullstory/babel-plugin-annotate-react plugin, edit your module rules for files matching `/\.(js|mjs|jsx|ts|tsx)$/` to include the plugin. In our sample repository we modified the `fullstory-babel-plugin-annotate-react/samples/single-page-app/config/webpack.config.js` file (around line 426). Since this sample is using the local plugin, we use `../..` as the plugin name. However, you should use the npm package name: `@fullstory/babel-plugin-annotate-react`.
```

Reply:
fixed!